scripts/build_npm.ts

```diff
@@ -4,14 +4,14 @@ import { build, emptyDir } from "./script_deps.ts";
  * This script builds the SDK for NPM.
  *
  * It is run from the root of the repository with:
- * 'Deno run -A scripts/build_npm.ts 1.0.0'
+ * 'deno run -A scripts/build_npm.ts 1.0.0'
  *
  * '1.0.0' is the version, or git tag, you wish to use.
  *
  * after running the script, the SDK can be published to NPM with:
- * 'cd npm && npm publish --access public'
+ * 'npm publish ./npm --access public'
  *
- * @example Deno run -A scripts/build_npm.ts 1.0.0 && cd npm && npm publish --access public
+ * @example deno run -A scripts/build_npm.ts 1.0.0 && npm publish ./npm --access public
```

Comment:
Is there a key in ./npm or something? Because keys should not be in particular repos but in the users .npmrc in the home directory, file if I remember correctly.

Reply:
This is just to avoid "cd npm"

Reply:
The scripts assume you are running from the project root + have npm already set up and authenticated.
We could improve this in the future